src/libs/ajax/DataRepo.ts

```diff
@@ -146,11 +164,11 @@ export interface DataRepoContract {
   };
   snapshot: (snapshotId: string) => {
     details: () => Promise<Snapshot>;
-    exportSnapshot: () => Promise<{}>;
+    exportSnapshot: () => Promise<JobModel>;
   };
   job: (jobId: string) => {
-    details: () => Promise<{}>;
-    result: () => Promise<{}>;
+    details: () => Promise<JobModel>;
+    result: () => Promise<any>;
```

Comment:
In this case the underlying object is json so `any` is the best type possible?

Reply:
Yes, and `any` is more unknown property friendly, as opposed to `{}`, which seems to assume an empty object? It errored at least with not knowing what the type was.

Reply:
That said, I think I can actually use `Object` which might be even better.